src/contexts/Theme.tsx

```diff
@@ -0,0 +1,50 @@
+import * as React from 'react'
+import { ThemeProvider } from '@mui/material/styles'
+import { dark, light } from '@/plugins/muiTheme'
+
+interface Children {
+  children: React.ReactNode
+}
+
+// interface ColorModeContextValue {
+//   mode: 'light' | 'dark'
+//   colorMode: {
+//     toggleColorMode: () => void
+//   }
+//   theme: Theme
+// }
+
+const ColorModeContext = React.createContext({ toggleColorMode: () => {} })
```

Comment:
for the context lets have this
```ts
const ColorModeContext = React.createContext<ColorModeContextValue>({
  toggleColorMode: () => {},
  mode: 'light'
})
```
this way the context depends on the interface

Reply:
Fixed